deploy/aws/boot.sh

```diff
@@ -68,6 +68,9 @@ do
   fi
 done
 
+ENCODED_mongo_root_user=$mongo_root_user
```

Comment:
```suggestion
encoded_mongo_root_user=$mongo_root_user
```

Please follow naming conventions. Why have all-caps in the variable name here?

Reply:
just to highlight the `encoded` credential.

Reply:
Converted to lower_case